azure/packages/azure-client/package.json

```diff
@@ -78,7 +78,7 @@
     "typescript": "~4.5.5"
   },
   "peerDependencies": {
-    "fluid-framework": "^1.0.1"
```

Comment:
why wouldn't you want minor versions? this will make it harder for developers, and doesn't match all the fluid framework  package dependencies which makes it really hard to manage

Reply:
for instance, fluid-framework depend on sequence "^1.0.1" which depends on mergetree "^1.0.1", pinning fluid framework will not pin those dependencies.

Reply:
what you probably want to do is move fluid-framework to azure client, and change all it's references to ~, then you will be pinning a lot more things, at least on the surface area. transitive dependencies like merge-tree above will still be allowed to change

Reply:
maybe it can be done where it is, and not move it to azure-client, but i'm not sure

Reply:
The goal here is to reduce the velocity of minor changes from fluid for Azure developers. This will allow us to validate these changes before releasing them. Our guidance to developers will be to use ~ (similar to many open source frameworks). If they choose ^ they will be able to do this with azure-client and the packages will still resolve correctly.

Reply:
making this ~ will not slow anything down though, as all the dependencies of fluid-framework are ^. that is the point i'm trying to make. this doesn't do anything.

Reply:
for example, we pin fluid-framework at ~1.0.1, it will still install latest of all of the dependencies, so if the latest releases of client packages are 1.17.0, then @fluidframework/sequence@1.17.0 will be installed.

Reply:
https://github.com/microsoft/FluidFramework/blob/11848cd181cad0729b656cf9e3e5a9fe11816942/packages/framework/fluid-framework/package.json#L42

Reply:
and even if you pin those, the dependencies of those dependencies will be unpinned. in general this probably increases risk for consumers, as we try to support mix and match, we don't test it well, so it's best if everything is aligned.